client/state/selectors/test/get-account-closure-sites.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
We've moved away from Chai in favour of Jest in newer tests. If you remove the Chai import, you can change your assertions from `to.eql` to `toEqual` and it should work :) You don't need to add any new imports for Jest.

Here's an example test suite:

https://github.com/Automattic/wp-calypso/blob/f2c9d99b07e14ea938829e9b2e5b31b6fb388a90/client/lib/format-number-compact/test/index.js

Reply:
Thanks, done! :)